lldb/include/lldb/Symbol/UnwindPlan.h

```diff
@@ -564,9 +557,8 @@ class UnwindPlan {
   }
 
 private:
-  typedef std::vector<RowSP> collection;
-  collection m_row_list;
-  AddressRange m_plan_valid_address_range;
+  std::map<lldb::addr_t, RowSP> m_rows;
```

Comment:
Would this benefit from being a DenseMap?

Reply:
It won't work, since the order of entries is important. What I can imagine is constructing an unwind plan (in lower case, i.e. not  `UnwindPlan`) in a map object, and then passing that to an UnwindPlan constructor which will flatten it in. It would be a slightly larger change, but I think the result could be nice as it would essentially make an UnwindPlan immutable after construction. (I like immutable data structures, and unwind plans really must be immutable as they are cached)